step-00/lib/main.dart

```diff
@@ -1,24 +1 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
 
-import 'package:flutter/cupertino.dart';
-import 'package:flutter/services.dart';
-import 'app.dart';
-
-void main() {
-  SystemChrome.setPreferredOrientations(
```

Comment:
Would this need accompanying narratives to explain what this is for?

Reply:
Fixed in https://github.com/googlecodelabs/flutter-cupertino-store/pull/6/files#diff-babe824401a3594961b3ce1a44c1baa9